src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/OpenIdController.cs

```diff
@@ -46,7 +46,7 @@ public class OpenIdController : ControllerBase
         [HttpGet("openid-configuration")]
         public async Task<IActionResult> GetOpenIdConfigurationAsync()
         {
-            string baseUrl = _generalSettings.PlatformEndpoint;
+            string baseUrl = _generalSettings.AltinnOidcIssuerUrl;
```

Comment:
Hvordan vil denne url se ut. Ser den ikke er satt i app settings

Reply:
It's now hardcoded in the `AltinnOidcIssuerUrl` property getter